services/app-api/src/resolvers/healthPlanPackage/submitHealthPlanPackage.ts

```diff
@@ -77,24 +76,19 @@ function submit(
         submittedAt: new Date(),
     }
 
+    // move this check into isValidContract when feature flag is removed
     const validPopulationCovered = featureFlags?.['chip-only-form']
         ? hasValidPopulationCoverage(
               maybeStateSubmission as LockedHealthPlanFormDataType
           )
         : true
 
-    const validRateCertAssurance = hasValidRateCertAssurance(
```

Comment:
^this check is now part of `isValidContract`. I think we should avoid standalone form field checks as much as possible in resolvers, try to lean on bigger functions like isValidContract or isValidRates that check entire sets of fields.  Ideally those bigger functions could also handle feature flags too (but that seems like a bigger refactor so I held off).